examples/tim1_pwm_complementary_outputs/.gdbinit

```diff
@@ -0,0 +1,2 @@
+file tim1_pwm.elf
+target extended-remote localhost:3333
```

Comment:
I don't believe this file should be committed.  cc @maxgerhardt

Reply:
OK, deleted file.